BooruSharp/Booru/ABooru.cs

```diff
@@ -152,8 +152,6 @@ protected ABooru(string baseUrl, UrlFormat format, BooruOptions options)
                 _imageUrlXml = _imageUrl.Replace("json=1", "json=0");
             else if (_format == UrlFormat.postIndexJson)
                 _imageUrlXml = _imageUrl.Replace("index.json", "index.xml");
-            else
```

Comment:
Why did you remove the else case?

Reply:
IMO it's redundant, since by default `_imageUrlXml` will be initialized to `null`.